src/Microsoft.Azure.SignalR/ServerConnections/ServiceConnection.Log.cs

```diff
@@ -27,6 +27,12 @@ private static class Log
             private static readonly Action<ILogger, Exception> _applicationComplete =
                 LoggerMessage.Define(LogLevel.Debug, new EventId(4, "ApplicationComplete"), "Application task completes.");
 
+            private static readonly Action<ILogger, int, string, string, Exception> _closingClientConnectionsFromInstance =
+                LoggerMessage.Define<int, string, string>(LogLevel.Information, new EventId(5, "ClosingClientConnectionsFromInstance"), "Closing {ClientCount} client connection(s) for server connection {ServerConnectionId} from Azure SignalR instance {InstanceId}.");
```

Comment:
how about reusing `_closingClientConnections`? After all customer is unable to get anything from {InstanceId}...